index.js

```diff
@@ -128,9 +128,10 @@ function init () {
   })
 
   req.addEventListener('error', function (e) {
-    document.querySelector('#proposals-count').innerText = 'Proposal data failed to load.'
+    document.querySelector('#proposals-count-number').innerText = 'Proposal data failed to load.'
   })
 
+  document.querySelector('#proposals-count-number').innerHTML = 'Loading ...';
```

Comment:
Remove the semicolon here and other places where they appear to follow the [style guide](https://standardjs.com).

Reply:
fixed